.github/workflows/check.yaml

```diff
@@ -2,10 +2,6 @@ name: Check Code
 
 on:
   push:
-
-  pull_request:
-    branches: [ main ]
```

Comment:
Yeah, since we already run the build on pushes to any branch, I noticed all the workflows occured twice on any PR.. I can't see any reason why we'd keep it that way unless we change the rule to not build for all pushes later on.